From: Leon Romanovsky <leonro@mellanox.com>
Date: Tue, 19 Jan 2016 11:11:24 +0200
Subject: IB/mlx5: Delete locally redefined variable
Patch-mainline: v4.5-rc1
Git-commit: 50ca6ed21e7c389c250ee30b0bb4df66157a1115
References: bsc#966170 FATE#320225 bsc#966172 FATE#320226

Fix the following sparse warning:
drivers/infiniband/hw/mlx5/main.c:1061:29: warning: symbol 'pfn' shadows
an earlier one
drivers/infiniband/hw/mlx5/main.c:1030:21: originally declared here

Fixes: d69e3bcf7976 ('IB/mlx5: Mmap the HCA's core clock register to user-space')
Signed-off-by: Leon Romanovsky <leonro@mellanox.com>
Signed-off-by: Doug Ledford <dledford@redhat.com>
Acked-by: Benjamin Poirier <bpoirier@suse.com>
---
 drivers/infiniband/hw/mlx5/main.c |    4 ----
 1 file changed, 4 deletions(-)

--- a/drivers/infiniband/hw/mlx5/main.c
+++ b/drivers/infiniband/hw/mlx5/main.c
@@ -1060,9 +1060,6 @@ static int mlx5_ib_mmap(struct ib_uconte
 		return -ENOSYS;
 
 	case MLX5_IB_MMAP_CORE_CLOCK:
-	{
-		phys_addr_t pfn;
-
 		if (vma->vm_end - vma->vm_start != PAGE_SIZE)
 			return -EINVAL;
 
@@ -1085,7 +1082,6 @@ static int mlx5_ib_mmap(struct ib_uconte
 			    vma->vm_start,
 			    (unsigned long long)pfn << PAGE_SHIFT);
 		break;
-	}
 
 	default:
 		return -EINVAL;
